fred2/fredview.cpp

```diff
@@ -526,9 +526,11 @@ int drag_objects()
 		Dup_drag = 0;
 
 	if (Dup_drag == 1) {
-		dup_object(NULL);  // reset waypoint list
 		cobj = Duped_wing = -1;
 		flag = 0;
+
+		int duping_waypoint_list = -1;
```

Comment:
Please also replicate these changes for qtFRED in `EditorViewport::drag_objects`.

Reply:
Ok.